webofneeds/won-cryptography/src/main/java/won/cryptography/rdfsign/WonKeysReaderWriter.java

```diff
@@ -158,23 +155,19 @@ private void readFromModel(final Model model, final Map<String, PublicKey> keys,
   public Set<String> readKeyReferences(Dataset dataset)
     throws NoSuchAlgorithmException, NoSuchProviderException, InvalidKeySpecException {
     Set<String> keyRefs = new HashSet<>();
-    readKeyReferences(dataset.getDefaultModel(), keyRefs);
     for (String name : RdfUtils.getModelNames(dataset)) {
-      readKeyReferences(dataset.getNamedModel(name), keyRefs);
+      readKeyReferences(dataset.getNamedModel(name), name, keyRefs);
     }
     return keyRefs;
   }
 
-  public void readKeyReferences(Model model, Set<String> keyRefs)
+  public void readKeyReferences(Model model, String modelName, Set<String> keyRefs)
     throws NoSuchAlgorithmException, NoSuchProviderException, InvalidKeySpecException {
-
```

Comment:
there is an unused parameter "modelName" in the readKeyReferences that should be used (?) or deleted if not used
i also dont see how this prevents adding key in default graph?